Mathlib/Computability/PDA.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2025 Stefan Hetzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tobias Leichtfried, Stefan Hetzl
+-/
+import Mathlib.Computability.Language
+
+/-!
+# Pushdown Automata
+
+This file contains the definition of a Pushdown Automaton (PDA). Pushdown automata recognize
+exactly the context-free languages.
+-/
+
+/-- A PDA is a set of states (`Q`), a tape alphabet (`T`), a stack alphabet (`S`), an initial state
+  (`initial_state`), a start symbol (`start_symbol`), a set of final states (`final_states`), a
+  transition function (`transition_fun`), and a proof that, for any input, the transition function
+  returns a finite set as output. -/
+structure PDA (Q T S : Type) [Fintype Q] [Fintype T] [Fintype S] where
+  /-- Initial state. -/
+  initial_state : Q
+  /-- Start symbol on the stack. -/
+  start_symbol : S
```

Comment:
Why is it "*initial* state", but "*start* symbol"?

Reply:
This is the terminology in one of the classics on the subject, the book our formalisation is based on: _Hopcroft, Ullman: Introduction to Automata Theory, Languages, and Computation, Addison-Wesley, 1979._ I am not sure about the root of this distinction.

Reply:
That is strange. Could you record this strangeness by adding a sentence in each of the two docstrings?